hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/conf/HddsConfServlet.java

```diff
@@ -165,6 +226,92 @@ private void processConfigTagRequest(HttpServletRequest request, String cmd,
 
   }
 
+  /**
+   * Build a map of property names to descriptions by reading from configuration resources.
+   * @param config the OzoneConfiguration to extract descriptions from
+   * @return map of property name to description
+   */
+  private Map<String, String> buildDescriptionMap(OzoneConfiguration config) {
+    Map<String, String> descriptionMap = new HashMap<>();
+    
+    // List of configuration resource names to check
+    String[] configResources = new String[] {
+        "core-default.xml",
+        "core-site.xml",
+        "hdfs-default.xml",
+        "hdfs-site.xml",
+        "hdds-common-default.xml",
+        "hdds-client-default.xml",
+        "hdds-container-service-default.xml",
+        "hdds-server-framework-default.xml",
+        "hdds-server-scm-default.xml",
+        "ozone-common-default.xml",
+        "ozone-csi-default.xml",
+        "ozone-manager-default.xml",
+        "ozone-recon-default.xml",
+        "ozone-default.xml",
+        "ozone-site.xml"
+    };
```

Comment:
Please define the list of resources in `OzoneConfiguration` and use it in both places, here and in:

https://github.com/apache/ozone/blob/3f54b1452c5b725431c69ca25e0bb2a3d400894e/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java#L213